tests/static/html/fluid.html

```diff
@@ -0,0 +1,32 @@
+<html>
```

Comment:
I intended to add a function to evaluate any arbitrary variable, but hadn't got to it.  I implemented this just now.  See the new file `browser-fluid-tests.js` for examples.